### ISSUE TITLE
How to open file with max window size

### ISSUE THREAD

**Issue by chris-770** [typora/typora-issues#6108]
On my laptop with win11, when I double click to open a md file, it can't be shown with max window size (as below), I have to click the "maximize window" in right corner to maximize the window.
Even when I manually click  "maximize window" for this file, typora can't remember next time when I open the same file.
<img width="1280" alt="2024-07-24 111844" src="https://github.com/user-attachments/assets/6dc06b42-ee43-40fb-aa60-8c940ddafdd2">

**Comment by weihongji** [comment]
Same issue here.

**Comment by weihongji** [comment]
My Solution:
1. Right-click on the shortcut of the Typora application and select "Properties".
    Example: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Typora
2. Select "Maximized" in the "Run" dropdown list.
3. Right-click the shortcut file again and select "Pin to taskbar". You'll get maximum window when launch Typora from taskbar.

**Comment by chris-770** [comment]
> My Solution:
> 
> 1. Right-click on the shortcut of the Typora application and select "Properties".
>    Example: C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Typora
> 2. Select "Maximized" in the "Run" dropdown list.

I've done this.

> 3. Right-click the shortcut file again and select "Pin to taskbar". You'll get maximum window when launch Typora from taskbar.

If I just launch Typora by click Typora icon (not double click a md file), I'll get maximum window. However, If I double click a md file, I can't get maximum window...

**Comment by weihongji** [comment]
Yeah, the issue still exists if Typora is started by openning a md file.

**Comment by KIssshotUnderHeart** [comment]
Same issue in 1.9.5
